client/extensions/woocommerce/state/ui/payments/methods/test/reducer.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import reducer from '../reducer';
+import {
+	openPaymentMethodForEdit,
+	closeEditingPaymentMethod,
+	cancelEditingPaymentMethod,
+	changePaymentMethodField,
+} from '../actions';
+
+const siteId = 123;
+
+const initialState = {
+	creates: [],
+	updates: [],
+	deletes: [],
+	currentlyEditingId: null,
+};
+
+describe( 'reducer', () => {
+	describe( 'openPaymentMethodForEdit', () => {
+		it( 'should set currentlyEditingId when the method has a server-side ID', () => {
+			const newState = reducer( initialState, openPaymentMethodForEdit( siteId, 1 ) );
```

Comment:
Yeah, I totally get what you mean.  I'd love to see an example.  I will address in another PR.